ui/src/worker/jobs/proxy.ts

```diff
@@ -1,26 +1,19 @@
+import Axios from 'axios'
+
 const proxy = async msg => {
   const {
     payload: {url, query, rp, db, uuid},
   } = msg
 
-  const response = await fetch(url, {
-    method: 'POST',
-    body: JSON.stringify({query, rp, db, uuid}),
-  })
-
-  const data = await response.json()
-
-  if (!response.ok) {
-    let message = 'proxy request failed'
-
-    if (data && data.message) {
-      message = data.message
-    }
-
-    return Promise.reject(message)
-  }
+  const body = {url, query, rp, db, uuid}
+  try{
```

Comment:
```suggestion
  try {
```

I wonder why the linter didn't fix the space